Sources/AudioKit/Internals/Manager/AKManager+SafeConnections.swift

```diff
@@ -1,168 +1,168 @@
-    ///         - progress: Closure called while rendering, use this to fetch render progress
-    ///
-    @available(iOS 11, macOS 10.13, tvOS 11, *)
-    public static func renderToFile(_ audioFile: AVAudioFile,
-                                    maximumFrameCount: AVAudioFrameCount = 4_096,
-                                    duration: Double,
-                                    prerender: (() -> Void)? = nil,
-                                    progress: ((Double) -> Void)? = nil) throws {
-
-        try engine.renderToFile(audioFile,
-                                maximumFrameCount: maximumFrameCount,
-                                duration: duration,
-                                prerender: prerender,
-                                progress: progress)
-    }
-
-    @available(iOS 11, macOS 10.13, tvOS 11, *)
-    public static func printConnections() {
-        let nodes: [AVAudioNode] = {
-            var nodes = Set<AVAudioNode>()
-            func addInputs(_ node: AVAudioNode) {
-                nodes.insert(node)
-                node.inputConnections().filter { $0.node != nil }.forEach {
-                    if let node = $0.node {
-                        addInputs(node)
-                    }
-                }
-            }
-            addInputs(engine.outputNode)
-            return Array(nodes)
-        }()
-
-        func nodeDescription(_ id: Int, _ node: AVAudioNode) -> String {
-            return "(\(id)]\(node.auAudioUnit.audioUnitName ?? String(describing: node))"
-        }
-
-        func formatDescription(_ format: AVAudioFormat) -> String {
-            guard let description = format.description.components(separatedBy: ":  ").dropFirst().first else {
-                return format.description
-            }
-            return "<" + description
-        }
-
-        let padLength = nodes.reduce(0) { max($0, nodeDescription(nodes.count, $1).count) }
-        func padded(_ string: String) -> String {
-            return string.count >= padLength ? string : string + String(repeating: " ", count: padLength - string.count)
-        }
-
-        nodes.enumerated().forEach { id, node in
-
-            let outputs: [(id: Int, node: AVAudioNode, bus: Int)] = node.connectionPoints.compactMap {
-                guard let node = $0.node, let id = nodes.firstIndex(of: node) else { return nil }
-                return (id, node, $0.bus)
-            }
-
-            let srcDescritption = padded(nodeDescription(id, node))
-            let format = formatDescription(node.outputFormat(forBus: 0))
-
-            outputs.forEach {
-                let dstDescription = nodeDescription($0.id, $0.node)
-                AKLog("\(srcDescritption) \(format) -> \(dstDescription)) bus: \($0.bus)")
-            }
-        }
-    }
-}
+//// Copyright AudioKit. All Rights Reserved. Revision History at http://github.com/AudioKit/AudioKit/
```

Comment:
Should we have all this commented out stuff when merging?

Reply:
I suppose not, will address.